hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineManagerImpl.java

```diff
@@ -697,6 +699,12 @@ public Map<String, Integer> getPipelineInfo() throws NotLeaderException {
     return pipelineInfo;
   }
 
+  @Override
+  public Map<String, String[]> getPipelineLeaders() throws NotLeaderException {
```

Comment:
`throws NotLeaderException` is redundant

Reply:
Thanks for the find, I have removed it !!